app/src/main/java/com/futurice/freesound/viewmodel/DisplayableItem.java

```diff
@@ -0,0 +1,46 @@
+package com.futurice.freesound.viewmodel;
```

Comment:
I would suggest that moving it into the `feature` package is the way to go. I'm struggling to think of a benefit of sharing (via a common package) a wrapped, generic data type; it doesn't do anything or explicitly express any useful information.